### DISCUSSION TITLE
Weld Type

### DISCUSSION THREAD

**Discussion by WOHughes** [idea-statica/ideastatica-public#689] [Rhino/Grasshopper] [answered]
Hi

Is it possible to apply butt welds through grasshopper? The option does not appear in the input list.

![image](https://github.com/user-attachments/assets/e4af8e89-3051-490f-938c-0b466f9e0a91)

Thanks!

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @WOHughes, Butt weld is equivilant to 4-Bevel in IOM/Grasshopper.

**Reply by nathanlukeIDEA** [reply]
P.S I noticed you have been specifying lengths as mm. All input needs to be in [Basic SI](https://developer.ideastatica.com/docs/iom/iom_units.html) units. We are looking at implementing better unit inputs.
